sensu/plugins/metrics-process-usage.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+#
+#  metrics-process-usage.py
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#  Python 2.7+ (untested on Python3, should work though)
+#  Python module: psutil https://pypi.python.org/pypi/psutil
+#
+# USAGE:
+#
+#  metrics-process-usage.py -n <process_name> -w <cpu_warning_pct> -c <cpu_critical_pct> -W <mem_warning_pct> -C <mem_critical_pct> [-s <graphite_scheme>]
+#
+# DESCRIPTION:
+# Finds the pid[s] corresponding to a process name and obtains the necessary
+# cpu and memory usage stats. Returns WARNING or CRITICAL when these stats
+# exceed user specified limits.
+#
+# Code adapted from Jaime Gogo's script in the Sensu Plugins community:
+# https://github.com/sensu-plugins/sensu-plugins-process-checks/blob/master/bin/metrics-per-process.py
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
```

Comment:
You point at the LICENSE file, but it only has Apache content. Maybe you should create a new MITLICENSE file to point to.